build.gradle

```diff
@@ -948,6 +948,7 @@ project(':clients') {
     compile libs.snappy
     compile libs.slf4jApi
 
+    compileOnly libs.slf4jlog4j
```

Comment:
Could we remove the `compileOnly libs.log4j` from the project('core') dependencies? (~line 680) I think we don't need it